sdk/maps/Azure.Maps.Rendering/api/Azure.Maps.Rendering.netstandard2.0.cs

```diff
@@ -234,6 +195,24 @@ public partial class MapTileSet
         public static bool operator !=(Azure.Maps.Rendering.MapTileSize left, Azure.Maps.Rendering.MapTileSize right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MediaType : System.IEquatable<Azure.Maps.Rendering.MediaType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public MediaType(string value) { throw null; }
+        public static Azure.Maps.Rendering.MediaType ImageJpeg { get { throw null; } }
+        public static Azure.Maps.Rendering.MediaType ImagePng { get { throw null; } }
+        public bool Equals(Azure.Maps.Rendering.MediaType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.Maps.Rendering.MediaType left, Azure.Maps.Rendering.MediaType right) { throw null; }
+        public static implicit operator Azure.Maps.Rendering.MediaType (string value) { throw null; }
+        public static bool operator !=(Azure.Maps.Rendering.MediaType left, Azure.Maps.Rendering.MediaType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
```

Comment:
We need to hide this, since it exists in the header and we're not using it anywhere

Reply:
We need to export `Accept` for users to set media type (and rename it to `AcceptMediaType` for better understanding).
[RenderingRestClient](https://github.com/JaneaSystems/azure-sdk-for-net/blob/090dc77d34ac8c18ca63afeb9f7afe3f5684977a/sdk/maps/Azure.Maps.Rendering/src/Generated/RenderRestClient.cs#L26) has `MediaType`, this should be exposed to users in `MapsRenderingClientOptions`